indexer/migrations/20230523_create_schema.sql

```diff
@@ -170,8 +208,8 @@ CREATE TABLE IF NOT EXISTS l1_bridge_deposits (
     -- Deposit information
 	from_address     VARCHAR NOT NULL,
 	to_address       VARCHAR NOT NULL,
-	l1_token_address VARCHAR NOT NULL,
-	l2_token_address VARCHAR NOT NULL,
+    l1_token_address VARCHAR NOT NULL, -- REFERENCES l1_tokens(token_address), uncomment me in future pr
+    l2_token_address VARCHAR NOT NULL, -- REFERENCES l2_tokens(token_address), uncomment me in future pr
```

Comment:
Don't want to break existing code.
Not sure where this indent comes from. Could add a sql formatter

Reply:
dude i was running into this same issue. The formatting would look all weird on GH versus locally